tests/jerry/date-tostring.js

```diff
@@ -88,3 +88,17 @@ catch (e)
   assert (e instanceof TypeError);
   assert (e.message === "Incompatible type");
 }
+
+assert (new Date (NaN).toJSON () == null);
+assert (new Date ("2015-07-16").toJSON () == "2015-07-16T00:00:00.000Z");
+assert (new Date ("2015-07-16T11:29:05.023").toJSON () == "2015-07-16T11:29:05.023Z");
+
```

Comment:
We should also add a test to see if the toJSON returns a string which can be processed by the Date constructor.